### PR TITLE
unnecessary pages removed, jqueryi18n removed and more!. Checkout out the code

### PR DESCRIPTION
**Pull request by JustinCast**

### CODE REVIEW

File: website/core/Footer.js [hunk 1]
```diff
@@ -5,67 +5,55 @@
...
-            <a href={this.docUrl('doc1.html', this.props.language)}>
-              Como Iniciar
-            </a>
-            <a href={this.docUrl('doc2.html', this.props.language)}>
-              Guías
-            </a>
-            <a href={this.docUrl('doc3.html', this.props.language)}>
-              APIs
+            <h5>Documentation</h5>
+            <a href={this.docUrl("doc1.html", this.props.language)}>
```
Comment (xavier506) [inline website/core/Footer.js]: please correct link target path

File: website/siteConfig.js [hunk 2]
```diff
@@ -65,11 +66,11 @@ const siteConfig = {
   separateCss: [
     "css/static_website.css"
   ],
-  // algolia: {
-  //   apiKey: '',
-  //   indexName: '',
-  //   algoliaOptions: {} // Optional, if provided by Algolia
-  // },
+  algolia: {
+    apiKey: 'f7e83f64ca037b5315b5ccd56f79910c',
```
Comment (xavier506) [inline website/siteConfig.js:70]: is it recommended by algolia to add this key here?

**Review by xavier506 — CHANGES_REQUESTED** [review]
@JustinCast 

please see my comments


great work ! 💯

File: website/siteConfig.js (hunk 2 above)
Comment (JustinCast) [reply website/siteConfig.js:70]: https://docusaurus.io/docs/en/search. In this section of the documentation, docusaurus give no warnings about the exposition of the api key.

**Pull request merged**